segmentation/utils/spatial_analysis_test.py

```diff
@@ -177,113 +172,129 @@ def make_expression_matrix(enrichment_type):
 
 def test_calculate_channel_spatial_enrichment():
 
+    dist_lim = 100
+
     excluded_colnames = ["cell_size", "Background", "HH3",
                          "summed_channel", "cellLabelInImage", "area",
                          "eccentricity", "major_axis_length", "minor_axis_length",
                          "perimeter", "SampleID", "FlowSOM_ID", "cell_type"]
 
     # Test z and p values
-
     marker_thresholds = make_threshold_mat()
 
-    # Positive enrichment
-    all_data_pos = make_expression_matrix("positive")
-    dist_mat_pos = make_distance_matrix("positive")
+    # Positive enrichment with direct matrix initialization
+    all_data_pos = make_expression_matrix(enrichment_type="positive")
+    dist_mat_pos_direct = make_distance_matrix(enrichment_type="positive", dist_lim=dist_lim)
 
-    values, stats = \
+    _, stats_pos = \
         spatial_analysis.calculate_channel_spatial_enrichment(
-            dist_mat_pos, marker_thresholds, all_data_pos,
-            excluded_colnames=excluded_colnames, bootstrap_num=100)
+            dist_mat_pos_direct, marker_thresholds, all_data_pos,
+            excluded_colnames=excluded_colnames, bootstrap_num=100,
+            dist_lim=dist_lim)
+
     # Test both Point8 and Point9
-    assert stats.loc["Point8", "p_pos", 2, 3] < .05
-    assert stats.loc["Point8", "p_neg", 2, 3] > .05
-    assert stats.loc["Point8", "z", 2, 3] > 0
-
-    assert stats.loc["Point9", "p_pos", 3, 2] < .05
-    assert stats.loc["Point9", "p_neg", 3, 2] > .05
-    assert stats.loc["Point9", "z", 3, 2] > 0
-    # Negative enrichment
+    assert stats_pos.loc["Point8", "p_pos", 2, 3] < .05
```

Comment:
Would be good to add some comments about what 3 and 2 are indexing to make this less cryptic to read

Reply:
Okay, these are the indices of the columns that contain the markers we care about: https://github.com/angelolab/segmentation/blob/4a7280af7dab2d83cbfe8b3eb5977bf00937275f/segmentation/utils/spatial_analysis_test.py#L86

Can you add a comment to that effect?